experiments/list_unsupported.sh

```diff
@@ -88,7 +87,21 @@ else
       exit 9
    fi
 fi
+
+# Can we build the support tool?
+# Problem: GNAT 2016 helpfully installs a g++ binary alongside gnat... except
+# it's way out of date so invoking g++ via PATH looking will pickup that gnat g++
+# compiler, rather than the default system compiler... Instead we need to
+# temporarily drop the GNAT tools off the path while we build the tool.
+saved_path="$PATH"
+export PATH=$(echo ${PATH} | tr ':' '\n' | grep -v "${ADA_HOME}" | paste -s -d : - )
```

Comment:
I think it'd be easier to search for system g++ via `command` before setting the PATH instead, and then just using the absolute path for this one command

Reply:
The problem is that PATH is set outside this script, not within the script, hence this attempt to try to remove the gnat path from PATH.